src/utils.ts

```diff
@@ -158,6 +164,7 @@ export default {
   sendTransaction,
   getScores,
   validateSchema,
+  getNumberWithOrdinal,
   getProvider,
```

Comment:
Can you export voting classes here too?
```suggestion
  getNumberWithOrdinal,
  voting,
  getProvider,
```